runner/entrypoint-dind-rootless.sh

```diff
@@ -22,6 +22,17 @@ jq ".\"registry-mirrors\"[0] = \"${DOCKER_REGISTRY_MIRROR}\"" /home/runner/.conf
 fi
 SCRIPT
 
+if [ -f /home/runner/.local ]; then
+  if [ ! -f /home/runner/.local/share ]; then
```

Comment:
shouldn't these be checks for directories? `-d`

Reply:
Good catch! Fixed in 300adcf7ec02b345e5ee173a24fde4f846089c75